aten/src/ATen/native/cpu/MultinomialKernel.cpp

```diff
@@ -123,7 +104,8 @@ void multinomial_apply(Tensor& result, const Tensor& self, const int64_t n_sampl
 
 static void multinomial_kernel_impl(Tensor& result, const Tensor& self, const int64_t n_sample, const bool with_replacement, Generator *gen) {
   AT_DISPATCH_FLOATING_TYPES(self.scalar_type(), "multinomial", [&] {
-    multinomial_apply<scalar_t>(result, self, n_sample, with_replacement, gen);
+    using accscalar_t = at::acc_type<scalar_t, false>;
```

Comment:
For boolean arguments it is nice to include a comment for what they are so that they're easier to read. i.e. `at::acc_type<scalar_t, /*is_cuda=*/false>`